main.go

```diff
@@ -78,6 +78,12 @@ func main() {
 		return
 	}
 
+	if len(vargs.Cluster) == 0 {
+		fmt.Println("Cluster: default")
```

Comment:
The cluster isn't set until `UpdateServiceInput` is called. If no cluster is passed in, the api chooses the cluster named `default`. `api.go` (ln 1526-8):

> 	// The short name or full Amazon Resource Name (ARN) of the cluster on which
	// to run your service. If you do not specify a cluster, the default cluster
	// is assumed.

There's no debug in the build output to inform the user when the default is used, so I think it is helpful in that respect.